frontend/src/sass/variables.sass

```diff
@@ -19,3 +19,4 @@ $footer-height: 7em
 $footer-height-narrow: 16em
 
 $narrow-screen: 900px
+$medium-screen: 1000px
```

Comment:
Tarvitseeko näitä olla kaksi? Toimisiko, jos narrow-screenin leveyden asettaisi 1000px:n?